src/main/webapp/contact.jsp

```diff
@@ -1,12 +1,15 @@
 <%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ page import="teammates.common.util.Config" %>
+<c:set var="supportEmail" value="<%= Config.SUPPORT_EMAIL %>" />
 <t:staticPage currentPage="contact">
     <h1 id="caption">
         Contact Us
     </h1>
     <img src="images/contact.png" width="125px" style="padding-left: 100px;">
     <div id="contentHolder">
         <p>
-            <span class="bold">Email: </span> You can contact us at the following email address - <a href="mailto:teammates@comp.nus.edu.sg">teammates@comp.nus.edu.sg</a>
+            <span class="bold">Email: </span> You can contact us at the following email address - <a href="mailto:${supportEmail}">${supportEmail}</a>
```

Comment:
Not sure if live server needs configuration due to this, but I'm leaving a note here just in case